apps/condo/lang/ru/messages/PASS_TICKET_COMMENT_CREATED/telegram.html.njk

```diff
@@ -0,0 +1 @@
+{{ message.meta.data.senderName }} (житель): {{ message.meta.data.comment }}
```

Comment:
We have a structure for telegram messages. I think it should be similar to the others

<img width="414" height="243" alt="image" src="https://github.com/user-attachments/assets/23941f54-3272-4dfa-adf4-e6a82a38c44a" />